bpdv3/Assets/Scripts/Core/Spawner.cs

```diff
@@ -90,6 +91,7 @@ private void SpawnObject()
             }
             objectPool.transform.position = SpawnPosition();
             objectPool.SetActive(true);
+            objectPool.GetComponent<Enemy>().hasSpawned = true;
```

Comment:
this boolean 'hasSpawned' was created to eliminate an error appearing on the console. Only created for this reason.